### PR TITLE
defaultworld: store the last world you were connected to

### PR DESCRIPTION
**Pull request by BenDol**
store the last world you were connected to and apply on startup.

### CODE REVIEW

**Comment by deathbeam** [comment]
This should probably be part of default world plugin, it is pretty unrelated to world hopper.

File: runelite-client/src/main/java/net/runelite/client/plugins/defaultworld/DefaultWorldPlugin.java [hunk 1]
```diff
@@ -54,6 +54,9 @@
 	@Inject
 	private DefaultWorldConfig config;
 
+	@Inject
+	private ConfigManager configManager;
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/defaultworld/DefaultWorldPlugin.java]: Just use hidden config property, like slayer plugin does

**Comment by BenDol** [comment]
ping

**Comment by Adam-** [comment]
This seems fine. I would probably name it `Use Last World` instead of `Use Last Connected World` since the phrasing there would probably confuse most people.

**Pull request merged**